lib/listener/helpers.js

```diff
@@ -13,6 +13,10 @@ module.exports = function () {
   const runHelpersHook = (hook, param) => {
     if (store.dryRun) return;
     Object.keys(helpers).forEach((key) => {
+      if (!helpers[key][hook]) {
+	    console.log(`${key}'s hook value is: ${helpers[key][hook]}`);
+	    return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4932579)